src/tools/coretool

```diff
@@ -806,6 +806,14 @@ class CoreFile:
         if not self.bitstream or not self.core_target or not self.bit_name or not self.bit_version:
             raise ValueError("need at least bitstream, target, name and version to build core")
 
+        # layout embedded files
+        embed_offset: int = 4096 + self.bitstream.raw_data_length
+        for file in self.embed_files:
+            file.offset = embed_offset
+            file.next = embed_offset + 40 + file.raw_data_length
+            embed_offset = file.next
+        embed_offset = self.embed_files[0].offset if self.embed_count else 0
```

Comment:
I think we can get rid of that, as it is not used after this, right?

Reply:
Ok, having the RHS in the parameter list would blow this even more up. Then it's ok the way it is.